package.json

```diff
@@ -16,15 +16,18 @@
     "fifo": "^2.3.0",
     "mic": "^2.1.1",
     "object.pick": "^1.2.0",
-    "pigpio": "^0.4.0",
-    "rpi-ws281x-native": "^0.8.1",
     "semaphore": "^1.0.5",
     "sleep": "^5.0.0",
     "temp": "^0.8.3",
     "twitter": "^1.5.0",
     "web-audio-api": "^0.2.2",
     "watson-developer-cloud": "^2.11.1"
   },
+  "optionalDependencies": {
+    "//": "pigpio and pi-ws281x-native are raspberry exclusive. To the project works in Linux it should be keeped as optional.",
```

Comment:
yeah... Linux is only for development purpose.
Ex: if I'm traveling I don't have to go with a RPi to developer code.

That items are related with that note at README.md:

> FOR DEVELOPMENT PURPOSE: to develop using TJLib is possible to test your conversation recipe in Ubuntu.
> Just verify if alsa-base alsa-utils are installed and set configuration.microphone config with card/device found at arecord -l